addons/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveAlarmCommandClass.java

```diff
@@ -450,7 +443,12 @@ public boolean getInitialised() {
      */
     public class ZWaveAlarmValueEvent extends ZWaveCommandClassValueEvent {
 
-        private AlarmType alarmType;
+        private final ReportType reportType;
+        // v1
+        private int alarmType;
```

Comment:
I'm not sure why you changed the V1 alarm type to an integer? I don't believe that the previous concept was wrong - do you though?

I would have thought that we keep the V1 concept as it was and just add the event concept. If you think otherwise, then let's discuss - of course, we're guessing to some degree without the docs.